### PR TITLE
The talk to a human option

### PR DESCRIPTION
**Pull request by adriaanm**

### CODE REVIEW

**Comment by adriaanm** [comment]
review by those now officially on the hook -- @gkossakowski, @retronym, @lrytz, @Ichoran, @dickwall 

/cc those who might want to feature as well: @som-snytt, @alex22, @xeno-by, @densh, @vladureche, @phaller, @heathermiller...

**Comment by retronym** [comment]
LGTM :information_desk_person:

**Comment by retronym** [comment]
I hereby self-nominate for the Emoji-of-the-week award.

**Comment by adriaanm** [comment]
You have my vote :ballot_box_with_check:

**Comment by som-snytt** [comment]
I have some trouble distinguishing it from :person_with_pouting_face: but maybe that's the point.

**Comment by VladUreche** [comment]
@retronym: +1 on the Emoji award :+1:
@adriaanm, can you add me with "specialization and the scaladoc tool"? Or should I do a PR separately?

**Comment by adriaanm** [comment]
@VladUreche: done. 

I'll squash once we're done adding everyone. I think you can edit in your browser to submit a PR to my PR, if you're so inclined.

**Comment by VladUreche** [comment]
Thanks @adriaanm. Next time I'll PR your PR :laughing:

File: README.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ For more information on building and developing the core of Scala, read on! Plea
...
+If you need some help with your PR at any time, please feel free to @-mention anyone from the list below, and we will do our best to help you out:
+
+ username     |      full name       | talk to me about...                               |
+--------------|----------------------|---------------------------------------------------| 
+ adriaanm     | Adriaan Moors        | anything (type checker, pattern matcher, CI)      |
+ gkossakowski | Grzegorz Kossakowski | infrastructure, incremental compilation, back-end |
+ retronym     | Jason Zaugg          | Java 8 lambdas, tricky bug detective work         |
+ Ichoran      | Rex Kerr             | the collections library                           |
+ lrytz        | Lukas Rytz           | optimizer, named & default arguments              |
+ dickwall     | Dick Wall            | process & documentation                           |
```
Comment (retronym) [inline README.md:21]: In the interest or demonstrating our humanity (or, as the case may be, unicornity), how about:

|  | username | talk to me about... |
| --- | --- | --- |
| ! | [`@adriaanm`](https://github.com/adriaanm) | anything (type checker, pattern matcher, CI) |
| ! | [`@gkossakowski`](https://github.com/gkossakowski) | infrastructure, incremental compilation, back-end |
| ! | [`@retronym`](https://github.com/retronym) | Java 8 lambdas, tricky bug detective work |
| ! | [`@Ichoran`](https://github.com/Ichoran) | the collections library, performance |
| ! | [`@lrytz`](https://github.com/lrytz) | optimizer, named & default arguments |

Comment (adriaanm) [inline README.md:21]: implemented over at https://github.com/adriaanm/scala/compare/talk-to-us

Comment (adriaanm) [inline README.md:21]: (Can't push -f to scala/scala, where the web ui created this branch for me. I'll open a new PR once we have all feedback consolidated.)

Comment (retronym) [inline README.md:21]: Perhaps add the catch-all `@scala/team-core-scala` to the list, too.

**Comment by densh** [comment]
I wouldn't mind being added with "parser, quasiaquotes and other string interpolators and/or macros in standard library" in the second column. :)

**Comment by xeno-by** [comment]
Same about me for "macros and reflection".

File: README.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ For more information on building and developing the core of Scala, read on! Plea
...
+# Get in touch!
+If you need some help with your PR at any time, please feel free to @-mention anyone from the list below, and we will do our best to help you out:
+
+ username     |      full name       | talk to me about...                               |
+--------------|----------------------|---------------------------------------------------| 
+ adriaanm     | Adriaan Moors        | anything (type checker, pattern matcher, CI)      |
+ gkossakowski | Grzegorz Kossakowski | infrastructure, incremental compilation, back-end |
+ retronym     | Jason Zaugg          | Java 8 lambdas, tricky bug detective work         |
+ Ichoran      | Rex Kerr             | the collections library                           |
+ lrytz        | Lukas Rytz           | optimizer, named & default arguments              |
```
Comment (dickwall) [inline README.md:20]: Is Lukas also the REPL guy now? Paul P is listed as optmization and REPL throughout many docs, I can update optimization to Lukas throughout, but who's gonna be the REPL rebel?

Comment (lrytz) [inline README.md:20]: I haven't done any work on the REPL, but I think also the others in the list haven't worked on it much :)

**Comment by adriaanm** [comment]
ok, thanks everyone! closing in favor of squashed PR

File: README.md (hunk 2 above)
Comment (som-snytt) [inline README.md:20]: I've tinkered with the REPL and also rewritten parts; but really it ought to be spun off.

You could imagine it gaining community momentum from sbt: someone just posted a (putative) REPL issue on the sbt repo.

Anyone taking REPL seriously wants to rewrite it, in whole or in part.

BTW, REPL stands for "Rewrite Every Paulp Lambda". And PAULP stands for "Please Allow Us Less Programming."

Comment (adriaanm) [inline README.md:20]: Yep, have you seen @lihaoyi's [ammonite repl](https://github.com/lihaoyi/ammonite#ammonite-repl)?

Comment (adriaanm) [inline README.md:20]: PS: I bow down to your acronym reverse-engineering skills. If I had more of the forward direction of those, I might still be in academia, coming up with clever grant names, which seems like an important part.

**Pull request closed without merging**